package.json

```diff
@@ -3,7 +3,7 @@
     "version": "3.0.0-alpha",
     "description": "An open source repository for the Speedometer benchmark.",
     "engines": {
-        "node": ">=18.13.0",
+        "node": ">=22.0.0",
```

Comment:
do we want to ensure that we stay on v22 by adding `< 23` as well? I'm a bit afraid that this will switch to v24 automatically and we'll get build errors. We can always change that later if this happens of course.

Reply:
I believe you need to run `npm i` to update `package-lock.json` as well.

Reply:
ah yeah... of course.
Let me add a CI check for that as well in a separate PR.

Reply:
I think `npm ci` does it for dependencies, it's disappointing it doesn't work for everything that can update it!